src/UI/Internal/SideBar.elm

```diff
@@ -52,14 +65,77 @@ desktopColumn cfg page menu =
         ]
 
 
-mobileDrawer :
+desktopNonPersistent : RenderConfig -> Element msg -> Menu msg -> Element msg
+desktopNonPersistent cfg page (Menu.Menu prop opt) =
+    let
+        overlayAlpha =
+            if prop.isExpanded then
+                0.5
+
+            else
+                0
+
+        overlay =
```

Comment:
Can this be moved to its own function? Looks to me that it's just complicating reading...